core/blockchain/impl/persistant_map_util.hpp

```diff
@@ -39,7 +40,7 @@ namespace kagome::blockchain {
   /**
    * Errors that might occur during work with storage
    */
-  enum class LevelDbRepositoryError { INVALID_KEY = 1 };
+  enum class KeyLookupError { INVALID_KEY = 1 };
```

Comment:
I think that conceptually it might be not only a key lookup error, so please either change the comment or the enum name. For clarity.